go.mod

```diff
@@ -8,6 +8,10 @@ exclude k8s.io/kubernetes v1.13.0
 // Client-go does not handle different versions of mergo due to some breaking changes - use the matching version
 replace github.com/imdario/mergo => github.com/imdario/mergo v0.3.5
 
+replace istio.io/api => github.com/howardjohn/api v0.0.0-20220321203251-8186a01edef9
```

Comment:
need to update?

Reply:
This is intentional. The plan is to get approval for this with a hold, then merge API and update this. I don't want to just merge API now in case this one takes a while (need approval from everyone), so this let's us test in CI